fixieai/cli/auth/commands.py

```diff
@@ -0,0 +1,31 @@
+import click
+
+from fixieai.cli.auth import oauth_flow
+from fixieai.cli.auth import user_config
+
+
+def validate_not_authed(ctx, param, force):
```

Comment:
Nits: Please be sure to add tests and docstrings to these functions when you're done.